spinn_front_end_common/interface/spinnaker.cfg

```diff
@@ -41,7 +41,7 @@ critical =
 reports_enabled = True
 write_energy_report = False
 write_text_specs = False
-write_router_reports = False
+write_router_reports = True
```

Comment:
If I remember correctly, this makes things very slow on big networks...

Reply:
moved